docs/cookbook/images/image-on-entity.rst

```diff
@@ -0,0 +1,376 @@
+How to add an image to an entity? (One-To-One association)
+==========================================================
+
+As an example this cookbook will **add an image to the payment method**.
+The example uses a mix of attributes, annotations and yaml for configuration, but you choose your preferred flavor.
+
+1. Create the image entity class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``getPaymentMethod`` and ``setPaymentMethod`` are optional and are wrappers for the untyped ``getOwner`` and ``setOwner`` methods.
+The type is set in the constructor to an unspecific value for convenience because it's not relevant in an one-to-one relationship.
```

Comment:
```suggestion
The image ``type`` is set in the constructor to an unspecific value for convenience because it's not relevant in a one-to-one relationship.
```